multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -173,3 +175,12 @@
 S3_PREFIX = "s3://"
 SOURCEPROMPT_URL = "https://automl-mm-bench.s3.amazonaws.com/few_shot/templates.zip"
 SOURCEPROMPT_SHA1 = "c25cdf3730ff96ab4859b72e18d46ff117b62bd6"
+
+# ner
+ENTITY_GROUP = "entity_group"
+START_OFFSET = "start"
+END_OFFSET = "end"
+TOKEN_WORD_MAPPING = "token_word_mapping"
+WORD_OFFSETS = "word_offsets"
+NER = "ner"
```

Comment:
If using `NER` as a model name, better to put it together with other model names in this file. Also, don't forget to put `NER` in `HF_MODELS`.

Reply:
done in this [commit](https://github.com/awslabs/autogluon/pull/2183/commits/07f37bb1e29bb5e37a807c5498052e166455b4d7)